region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -137,6 +137,55 @@
         "If any Sidehoppers remain after using a Power Bomb, safely kill them from the hole below."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway (Morph Ball Dodge)",
+      "requires": [
+        {"obstaclesNotCleared": ["B"]},
+        "Morph"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 17,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "Open the door in advance, then lure the Sidehoopers to the left corner with morph. Wait for their jumps to be mostly synchronized.",
+        "Roll right as they start a jump, then quickly unmorph and start running right.",
+        "With good timing, Samus should be able to start running a couple tiles left of the leftmost Yapping Maw."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway (Tank the Damage)",
+      "requires": [
+        {"obstaclesNotCleared": ["B"]},
```

Comment:
```suggestion
        "canNeutralDamageBoost",
        "canTrickyJump",
        {"obstaclesNotCleared": ["B"]},
```

Reply:
Where does the neutral damage boost come into play?  I assume tricky jump is just to raise the difficulty.

Reply:
I use the hopper to push me into the corner, so I can get a full runway

Reply:
That seems fine, tho not required?  `canTrickyJump` seems problematic tho, this strat is really not very hard.  You're just tanking the damage to get runway.

Maybe instead we could use canTrickyJump to adjust the total number of hits you take?

Reply:
Do you think it would be good to redefine and rename `canIframeSpikeJump` to account for this type of strat?

Reply:
Oooh, I do really like `canHitbox` here!

Reply:
@kjbranch I could see it.  iFrameSpikeJump and this strat are about understanding Samus' damage iframes.  canHitBox is about enemy iframes.